provider/core/src/lib.rs

```diff
@@ -199,13 +199,13 @@ pub mod prelude {
 pub use prelude::*;
 
 // Less important non-prelude items
+pub use crate::any::MaybeSendSync;
```

Comment:
Issue: If you want to rename the trait, please do this in its own standalone PR so we can merge it before 1.0. Otherwise we should keep the current trait name.

Reply:
I want to do this whole PR before 1.0